content/docs/command-reference/remote/modify.md

```diff
@@ -182,6 +182,14 @@ these parameters, you could use the following options.
   ```dvc
   $ dvc remote modify myremote use_ssl false
   ```
+  
+- `ssl_verify` - whether or not to verify SSL certificates. By default SSL certificates are verified. You can provide the following values:
+  False - do not validate SSL certificates. SSL will still be used (unless use_ssl is False), but SSL certificates will not be verified.
+  path/to/cert/bundle.pem - A filename of the CA cert bundle to use.
```

Comment:
Please wrap to 80 chars 🙏

Reply:
And let us know when #5732 is approved in order to do a final check here 🙂

Reply:
@jorgeorpinel [#5732](https://github.com/iterative/dvc/pull/5732) has been merged.